src/main/external-resources/renderers/Samsung-TV-2021-0.conf

```diff
@@ -0,0 +1,115 @@
+#----------------------------------------------------------------------------
+# Profile for Samsung 2021 TVs.
+# QLED TV except for 43Q7*A/50Q7*A model, Q6*A and Q5*A Series/The Frame
+# NEO QLED 4K as well
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Samsung 2021 QLED TV
+RendererIcon = samsung-tv.png
+
+# ============================================================================
+# Manual:
+# https://downloadcenter.samsung.com/content/UM/202109/20210910124253866/OSNDVBEUA-7.2.0_EM_OSCAR_EU_ENG_210910.0.pdf
+# starting page 157 (specs on page 160)
+# ============================================================================
+#
+# As friendlyName and USER-AGENT can be changed in the TV menu, we can't rely on it
+# we will rely on modelName
+# modelName=QE75Q80AATXXC / QE50QN90AATXXC
+# ============================================================================
+#
+# try user agent even if most of time it's changed by user.
+UserAgentSearch = SEC_HHP_.*(Q[NEA]\d{2}Q[789]\d{1}A|Q[NEA]\d{2}QN[789]\d{1}A)
+UpnpDetailsSearch = Q[NEA]\d{2}Q[789]\d{1}A|Q[NEA]\d{2}QN[789]\d{1}A
+LoadingPriority = 5
+
+TranscodeVideo = MPEGTS-H265-AAC
```

Comment:
For most people's hardware it's a better idea to do `MPEGTS-H264-AAC` or `MPEGTS-H264-AC3` because it is more likely to handle realtime encoding

Reply:
I was following the Samsung recommendations (H264 is not 8K capable).

But as TV handle all 8K codec it should not be a problem to lower to H264.